README.md

```diff
@@ -28,22 +27,67 @@ http://localhost:19000
 
 > login / password: **admin** / **admin**
 
-<h3 align="center">Requirements</h3>
+<h3>Collect feedback on your model</h3>
+  
+  <a align='center' href="https://app.arcade.software/share/zPfpXFt8lRm5XDkaWM85">
+         <img width='600' alt="Interactive demo" src="readme/feedback.png">
+      </a>
+<br/>
+  <br/>
+  <br/>
+  
+  * Collaborate with instant feedback.<br/>
+  * Improve ML models with business stakeholders in no time.
+  
+ <h3>Create Tests from the feedback</h3>
```

Comment:
Can you reuse the same title as on the website, "Get automated test suites in seconds" ?

Reply:
Nitpick: can you remove the final period "." in the end of the title? Details matter.

Reply:
Done